spec/fizz_buzz_spec.rb

```diff
@@ -1 +1,48 @@
 require "fizz_buzz"
+
+describe FizzBuzz do
+
+  describe ".divisible_by" do
+
```

Comment:
since ALL of the tests are within this describe block you aren't adding any more information.  You can leave it out and just have the context.

Reply:
Also, if you want to indicate the tests are going to test a function e.g describe_by then use a # not a .  e.g. "#divisible_by"  or even "#FizzBuzz#divisible_by"